### PR TITLE
Leave application apis

### PR DESCRIPTION
**Pull request by surekhag**

### CODE REVIEW

File: server/app/api/controllers/project_allocation.js [hunk 1]
```diff
@@ -84,14 +86,15 @@ module.exports = {
   delete: function (req, res, next) {
     Project_Allocation_Model.findOneAndUpdate({ _id: req.params.id },
       {
-        status: "Inactive"
+        status: "Inactive",
+        updated_date : new Date(),
+        last_updated_by : req.user.userName
       },
-      function (err, userInfo) {
+      function (err, info) {
```
Comment (bhushan-oe) [inline server/app/api/controllers/project_allocation.js]: info is not being used

File: server/app/api/controllers/peer_review.js [hunk 2]
```diff
@@ -95,11 +94,12 @@ module.exports = {
     Peer_Review_Model.findOneAndUpdate(
       { _id: req.params.id },
       {
-        status: "Inactive"
+        status: "Inactive",
+        updated_date : new Date(),
+        last_updated_by : req.user.userName,
       },
-      function (err, userInfo) {
+      function (err, info) {
```
Comment (bhushan-oe) [inline server/app/api/controllers/peer_review.js]: info is not being used

File: server/app/api/controllers/leaves_application.js [hunk 3]
```diff
@@ -41,16 +41,80 @@ module.exports = {
...
+     },
+
+  delete: function (req, res, next) {
+    Leaves_Application_Model.findOneAndUpdate({ _id: req.params.id },
+      {
+        status: "Inactive",
+        updated_date : new Date(),
+        last_updated_by : req.user.userName
+      },
+      function (err,info) {
```
Comment (bhushan-oe) [inline server/app/api/controllers/leaves_application.js]: info is not being used

File: server/app/api/controllers/self_review.js [hunk 4]
```diff
@@ -123,11 +122,12 @@ module.exports = {
     Self_Review_Model.findOneAndUpdate(
       { _id: req.params.id },
       {
-        status: "Inactive"
+        status: "Inactive",
+        updated_date : new Date(),
+        last_updated_by : req.user.userName
       },
-      function (err, userInfo) {
+      function (err, info) {
```
Comment (bhushan-oe) [inline server/app/api/controllers/self_review.js]: info

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: server/app/api/controllers/peer_review.js (hunk 2 above)
Comment (surekhag) [reply server/app/api/controllers/peer_review.js]: Info was assumed as part of syntax as call back function. Removed as it's not being used and api is working fine without info parameter.

File: server/app/api/controllers/project_allocation.js (hunk 1 above)
Comment (surekhag) [reply server/app/api/controllers/project_allocation.js]: Info was assumed as part of syntax as call back function. Removed as it's not being used and api is working fine without info parameter.

File: server/app/api/controllers/leaves_application.js (hunk 3 above)
Comment (surekhag) [reply server/app/api/controllers/leaves_application.js]: Info was assumed as part of syntax as call back function. Removed as it's not being used and api is working fine without info parameter.

File: server/app/api/controllers/self_review.js (hunk 4 above)
Comment (surekhag) [reply server/app/api/controllers/self_review.js]: Info was assumed as part of syntax as call back function. Removed as it's not being used and api is working fine without info parameter.

Comment (surekhag) [reply server/app/api/controllers/self_review.js]: Updated same change for all working api's

**Review by bhushan-oe — APPROVED** [review]

**Pull request merged**